mote/projects/base/library/browser/patterns/atoms/picture/element.html

```diff
@@ -4,20 +4,23 @@
     {% if classes %}
         class="{% for v in classes.values %}{{ v }} {% endfor %}"
     {% endif %}
+    {% for k, v in attrs.items %}
+        {{ k }}="{{ v }}"
+    {% endfor %}
     >
     <!--[if IE 9><video style="display: none;"><![endif]-->
     {% for source in sources %}
         <source
             srcset="
-                {% for di in source.srcsets %}
+                {% for di in source.srcset.images %}
```

Comment:
The changes to this pattern are one piece of backwards compatibility I am going to have to insist on.

I neglected the fact that every `<source>` tag can have its own attributes, whereas before it was just a list of images we iterated over.

This corrects that oversight, and now this pattern fully conforms to the responsive image spec. Probably.